src/main/java/org/opensearch/security/authtoken/jwt/JwtVendor.java

```diff
@@ -105,13 +105,16 @@ public String createJwt(
         List<String> roles,
         List<String> backendRoles
     ) throws Exception {
+        String tokenIdentifier = "OBO";
         long timeMillis = timeProvider.getAsLong();
         Instant now = Instant.ofEpochMilli(timeProvider.getAsLong());
 
         jwtProducer.setSignatureProvider(JwsUtils.getSignatureProvider(signingKey));
         JwtClaims jwtClaims = new JwtClaims();
         JwtToken jwt = new JwtToken(jwtClaims);
 
+        jwtClaims.setProperty("token_identifier", tokenIdentifier);
```

Comment:
Maybe `typ` here? Short for type. https://datatracker.ietf.org/doc/html/rfc7519#section-5.1

From my understanding `typ` is typically used to help the receiver of this token decide what logic to use while handling.

Reply:
You mean the `SetProperty("typ", xxx)`? I didnt see there is function for `setType()`, but I do see there is something like `setTokenId`. I'm not sure if the id should fit into our case?

Reply:
Thanks for the reference btw. Yes I have change the key into `typ` with the `setProperty()` still.